exec/java-exec/src/test/java/org/apache/drill/TestBugFixes.java

```diff
@@ -316,4 +316,9 @@ public void testDRILL6318() throws Exception {
     rows = testSql("SELECT FLATTEN(data) AS d FROM cp.`jsoninput/bug6318.json` LIMIT 3 OFFSET 5");
     Assert.assertEquals(3, rows);
   }
+
+  @Test
+  public void testDRILL7622() throws Exception {
```

Comment:
Please move this test into `org.apache.drill.exec.fn.impl.TestAggregateFunctions` class and update its name to describe the issue it checks.